.gitignore

```diff
@@ -5,3 +5,5 @@ dist/
 dist-newstyle/
 cabal.project.local
 
+*.swp
```

Comment:
Would recommend to add it to your personal user gitignore, but I don't mind having a copy here.

Reply:
Cheers. I haven't set that up yet. Still setting up home-manager.